ui/desktop/src/main.ts

```diff
@@ -530,6 +530,10 @@ app.whenReady().then(async () => {
   });
 
   ipcMain.on('create-chat-window', (_, query, dir, version) => {
+    if (!dir?.trim()) {
```

Comment:
This checks for `undefined`, `null`, and `empty strings`. If we want to default `empty strings` to the root directory I can go ahead and update this. Will defer to the Goose team.